libchisel/Cargo.toml

```diff
@@ -11,3 +11,4 @@ homepage = "https://github.com/wasmx/wasm-chisel"
 parity-wasm = "0.35.2"
 rustc-hex = "1.0"
 byteorder = "1.2.4"
+binaryen = "0.5.0"
```

Comment:
It doesn't seem like this crate has been maintained since mid-October? cc @pepyakin

Reply:
I would say that the crate is in lazily maintanance mode : )
I add features (like wasm-opt) whenever I need them and approve PR whenever they go my way : )

I'm not sure myself if it's a good idea to use this crate here. On one hand, binaryen C API is not super convenient (it has global state). On the other hand, the other option is to invoke it as a command-line process...

Reply:
I actually have like 4 PRs I'll open shortly which fixes a lot of things.